hazelcast/src/main/java/com/hazelcast/datalink/HazelcastDataLink.java

```diff
@@ -45,15 +42,25 @@
 public class HazelcastDataLink extends DataLinkBase {
 
     /**
-     * The constant to be used as property key for XML
+     * The constant to be used as property key for XML string for connecting to remote cluster
      */
     public static final String CLIENT_XML = "client_xml";
 
     /**
-     * The constant to be used as property key for YAML
+     * The constant to be used as property key for YAML string for connecting to remote cluster
      */
     public static final String CLIENT_YML = "client_yml";
 
+    /**
+     * The constant to be used as property key for XML file path for connecting to remote cluster
+     */
+    public static final String CLIENT_XML_PATH = "client_xml_path";
+
+    /**
+     * The constant to be used as property key for YAML file path for connecting to remote cluster
+     */
+    public static final String CLIENT_YAML_PATH = "client_yaml_path";
```

Comment:
```suggestion
    public static final String CLIENT_YML_PATH = "client_yml_path";
```
We should have consistent naming